### ISSUE TITLE
Use 64bit integers for stats

### ISSUE THREAD

**Issue by stakach** [crystal-community/hardware#21]
Getting an error on an ubuntu server when using the shard

```
root@dev:/home/steve# cat /proc/stat
cpu  54487502 66080 20918747 3521385289 684532 0 4003818 0 0 0
cpu0 13713324 15897 5250057 880320209 127639 0 1352684 0 0 0
cpu1 13657940 17708 5226039 880393773 131338 0 1089484 0 0 0
cpu2 13566777 16925 5235667 880447071 134123 0 884527 0 0 0
cpu3 13549459 15548 5206983 880224234 291431 0 677121 0 0 0
```

The CPU numbers such as `3521385289` are larger than 32 bits so hit an issue when parsing this
https://github.com/crystal-community/hardware/blob/master/src/cpu.cr#L35

**Comment by j8r** [comment]
Ok for me to change to `Int64`, that's already the case for `Memory`.
Do you want to open a PR, or I can do it.

**Comment by j8r** [comment]
Closed with #22
